core_data_modules/traced_data/io.py

```diff
@@ -187,7 +191,44 @@ def export_traced_data_iterable_to_coda_with_scheme(data, key_of_raw, key_of_cod
         # Export each message to a row in Coda's datafile format.
         scheme_id = "1"
         code_ids = dict()  # of code -> code id
-        for i, td in enumerate(unique_data):
+        i = 0
+
+        if prev_f is not None:
+            # Read the previously coded Coda file
+            prev_rows = list(csv.DictReader(prev_f, delimiter=";"))
+            
+            # Exclude items in unique_data which are in the previously coded file.
+            prev_data = set(map(lambda row: row["data"], prev_rows))
+            unique_data = list(filter(lambda td: td[key_of_raw] not in prev_data, unique_data))
+
+            # Detect scheme_id from the previously coded file, and ensure that the previous file had only one scheme id.
+            # TODO: Update this function to support importing multiple coding schemes.
```

Comment:
Is there an issue for this? If not, let's create one and cross-link it to here

Reply:
There is a fix :)

https://github.com/AfricasVoices/CoreDataModules/pull/46